ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/jackson/mixin/RepresentativeMixIn.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.cmc.ccd.jackson.mixin;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonUnwrapped;
+import uk.gov.hmcts.cmc.domain.models.Address;
+import uk.gov.hmcts.cmc.domain.models.legalrep.ContactDetails;
+
+@SuppressWarnings("squid:S1610")
+public abstract class RepresentativeMixIn {
+
+    @JsonProperty("representativeOrganisationName")
+    abstract String getOrganisationName();
+
+    @JsonProperty("representativeOrganisationAddress")
+    abstract Address getOrganisationAddress();
+
+    @JsonUnwrapped(prefix = "representativeOrganisation")
```

Comment:
Is this capitalising correcting when applying prefix? I means is it generating `representativeOrganisationphone` or `representativeOrganisationPhone` ?